File: primetest.java
public class primetest {
    private static int NUM_PRIMES = 500000;
    public static void main(String args[]) {
        int array[] = new int[NUM_PRIMES];
        boolean isPrime;
        long t1, t2;
        double elapsedMsec;

        t1 = System.nanoTime();
        int i = 2;
        for (int idx = 0; idx < NUM_PRIMES; ++i) {
            isPrime = true;
            for (int y = 0; y < idx; ++y) {
                if (i % array[y] == 0) {
                    isPrime = false;
                    break;
                }
            }
            if (isPrime) {
                array[idx] = i;
                ++idx;
            }
        }
        t2 = System.nanoTime();
        for(int x = 0; x < 1000; ++x) {
            System.out.print(array[x] + "\t");
        }
        elapsedMsec = (double)(t2 - t1) / 1000;
        System.out.println("\nTime needed for process: " + elapsedMsec + " Milliseconds.");

    }
}

File: primetest.py
import time

NUM_PRIMES = 50000
array = list(range(0, NUM_PRIMES))
isPrime = True

t1 = time.time()
i = 2
idx = 0
while (idx < NUM_PRIMES):
    isPrime = True;
    y = 0
    for y in range(0, idx):
        if (i % array[y] == 0):
            isPrime = False
            break
    if (isPrime):
        array[idx] = i
        idx += 1
    i += 1
t2 = time.time()
for x in range(0, 100):
    print(array[x], end="\t")

elapsedMsec = (t2 - t1) * 1000.0;
print("\nTime needed for process:", elapsedMsec, "Milliseconds.");
input()
